src/types/redis_bitmap.cc

```diff
@@ -502,7 +503,7 @@ rocksdb::Status Bitmap::BitOp(BitOpFlags op_flag, const std::string &op_name, co
 
         if (op_flag == kBitOpNot) {
           if (frag_index == stop_index) {
-            frag_maxlen = max_size % kBitmapSegmentBytes;
+            frag_maxlen = max_size % (kBitmapSegmentBytes + 1);
```

Comment:
@git-hulk when `max_size == kBitmapSegmentBytes * n`. the last segment would be cleared as 0 here.

Reply:
Please also add a comment to explicitly explain the meaning of this 1